lib/rails_cloud_tasks/rack/jobs.rb

```diff
@@ -0,0 +1,47 @@
+require 'json'
+require 'rack'
+
+module RailsCloudTasks
+  module Rack
+    class Jobs
+      class << self
+        def call(env)
+          job_class = extract_job_class(env)
+
+          request = ::Rack::Request.new(env)
+          job_args = extract_args(request)
+          job_class.perform_now(job_args)
+
+          response(200, { error: nil })
+        rescue Rack::UnregisteredJobError, Rack::InvalidPayloadError => e
+          response(422, { error: e.message })
+        rescue StandardError => e
+          response(500, { error: e.message })
+        end
+
+        private
+
+        def extract_job_class(env)
+          job_class = env.dig('action_dispatch.request.path_parameters', :job_class)
+
+          unless job_class.in?(RailsCloudTasks.config.jobs)
```

Comment:
Is it really necessary?

Reply:
I think so, as a security measure. You may not want to allow any of your jobs to be executed remotely.
Since there is no validation of who is requesting, this can restrict what can and cannot be run

Reply:
But there's no way to run any Job on Google Cloud Task but remotely. Why do you have a base code that could not run?

Reply:
Ok, removed